templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx

```diff
@@ -46,15 +45,24 @@ export function CreateFungibleAsset() {
       name &&
       symbol &&
       maxSupply &&
-      costToMint &&
+      mintFeePerFA &&
       decimal &&
       iconURL &&
       projectURL &&
       maxMintPerAccount
     ) {
       setDisableCreateAssetButton(false);
     }
-  }, [name, symbol, maxSupply, costToMint, decimal, iconURL, projectURL]);
+  }, [
+    name,
+    symbol,
+    maxSupply,
+    mintFeePerFA,
```

Comment:
is "mint fee per fa" / "cost to mint" a required field? can i create a FA that is free to mint?

Reply:
you can set it to 0 to make it free

Reply:
i see. then it probably shouldn't be required and the contract can set it if it is undefined. not a blocker tho